OTAnalytics/plugin_prototypes/track_visualization/track_viz.py

```diff
@@ -248,15 +248,15 @@ def _filter_tracks(
         Returns:
             DataFrame: filtered by classes, time and number of images
         """
-        track_df[
+        filtered_df = track_df[
```

Comment:
Because of the missing assignment of the `track_df` since the operation is not inplace, the tracks were not filtered to begin with. I fixed that here.